src/test/java/cloud/eppo/helpers/TestUtils.java

```diff
@@ -0,0 +1,62 @@
+package cloud.eppo.helpers;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.*;
+
+import cloud.eppo.BaseEppoClient;
+import cloud.eppo.EppoHttpClient;
+import cloud.eppo.EppoHttpClientRequestCallback;
+import java.lang.reflect.Field;
+import okhttp3.*;
+
+public class TestUtils {
```

Comment:
super minor nit: good practice to make "utils"-like classes `final` to tell users they're not supposed to subclass it